<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>264</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>199</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>n(u,a)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="function_type">
      <item>
       <property name="text">
        <string>a</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>a1+a2*(u-u')</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>a1+a2*u1+a2*u2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>nu+a1(u1-u1')+a2(u2-u2')</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>160</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Dпом.</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="hid_disp"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>160</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Iter</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="iter_count">
      <property name="maximum">
       <number>200</number>
      </property>
      <property name="value">
       <number>50</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>160</y>
     <width>160</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Interval</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="interval_len">
      <property name="maximum">
       <number>500</number>
      </property>
      <property name="value">
       <number>80</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>60</y>
     <width>160</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Graph1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Graph</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>200</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>calc</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
